-/index.md

```diff
@@ -1,6 +1,15 @@
 [Code Chica](./) is our free program for 8th–12th grade girls to gain knowledge
 about HTML, CSS, and JavaScript to create their own website.
 
+
+<div>
+  <a href="./wireframes.html">Wireframes</a>
+  <a href="./html.html">HTML</a>
+  <a href="./css.html">CSS</a>
+  <a href="./javascript.html">Javascript</a>
+</div>
```

Comment:
Do we want to wait before we add these links till there is more content in the pages themselves? I realized for the open issue, the right place for that beginner/code-chica content is actually going to be this html file you're adding in this PR

Reply:
please skip over the Code Chica part of the website...I am working on this...S-